### PR TITLE
[driver] upgrade to the latest version of vm_service_client

### PR DESCRIPTION
**Pull request by yjbanov**
Bonus:
- sort imports
- use consistent quotes in `pubspec.yaml`

### CODE REVIEW

File: packages/flutter_driver/pubspec.yaml [hunk 1]
```diff
@@ -8,17 +8,18 @@ environment:
...
-  file: ^0.1.0
-  json_rpc_2: any
+  file: '^0.1.0'
+  json_rpc_2: '^2.0.0'
   matcher: '>=0.12.0 <1.0.0'
-  path: ^1.3.0
-  vm_service_client: '>=0.1.2 <0.2.0'
+  path: '^1.3.0'
+  web_socket_channel: '^1.0.0'
+  vm_service_client: '>=0.2.0 <1.0.0'
```
Comment (devoncarew) [inline packages/flutter_driver/pubspec.yaml]: Shouldn't we be `^0.2.0` here? 0.3.0 might break us.

Comment (yjbanov) [inline packages/flutter_driver/pubspec.yaml]: Now that I learned what `^` means, yes. Fixed :)

**Comment by devoncarew** [comment]
lgtm

**Comment by yjbanov** [comment]
Thanks for quick review!

**Pull request merged**